<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">Qlabel{
text-color :qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
    </property>
    <property name="text">
     <string>Gestion Des Produits</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>731</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QTableView" name="tabProduit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>511</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="p_recherche">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="placeholderText">
       <string>Recherche ...</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>3</y>
        <width>201</width>
        <height>331</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_4">
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>10</y>
          <width>181</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Afficher Formulaire d'ajout :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>160</width>
          <height>178</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="p_ref">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Reference</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="p_nom">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Nom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="p_quantite">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Quantite</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="p_prix">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Prix</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="p_article">
           <property name="font">
            <font>
             <family>Impact</family>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Article id</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="p_ajouterBtn">
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>53</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Impact</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>AJOUT -  PRODUIT - FORMULAIRE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="annuler_ajout">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>290</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Impact</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MODIFICATION - SUPPRESSION - PRODUIT</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>162</width>
        <height>178</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="p_ref_m">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Reference</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="p_nom_m">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Nom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="p_quantite_m">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Quantite</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="p_prix_m">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Prix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="p_article_m">
         <property name="font">
          <font>
           <family>Impact</family>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Article id</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="p_modifierBtn">
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="p_supprimerBtn">
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="annuler_modification">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuproduit">
    <property name="title">
     <string>produit</string>
    </property>
   </widget>
   <addaction name="menuproduit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
